config/500-condition.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 The Knative Authors
```

Comment:
im gonna start out this review with the _smallest_ nitpick (shout out to @EliZucker) - should be 2019 Tekton Authors (we really need to add some kinda linting for that!)

Reply:
https://github.com/tektoncd/pipeline/issues/1107

Reply:
Ha! Good catch.